config.py.sample

```diff
@@ -37,6 +37,9 @@ MUSICBRAINZ_CLIENT_SECRET = "CLIENT_SECRET"
 LASTFM_API_KEY = "USE_LASTFM_API_KEY"
 
 
+# Max time in seconds after which the playing_now stream will expire.
+PLAYING_NOW_MAX_DURATION = 5 * 60
```

Comment:
We discussed this in IRC, and a little in the ticket: http://tickets.musicbrainz.org/browse/LB-80
Our other options for the length are
* Use a track length if it's submitted with the listen
* get the length from MB if we have a mbid
* otherwise use some fallback

We should remove the existing now playing if we add another one (since it's a single redis key, it should just get replaced), but we should also remove it if we get a listen submitted with a time within 80 minutes.
Honestly, I think 80 minutes is not a good default. What about the average of all our track lengths in musicbrainz + 1 standard deviation? (although, I have a hunch that that'll be around 5 minutes anyway). We're going to be "more correct" more of the time with a < 10 minute default than with 80.